cast/tests/e2e/helpers/fixtures.rs

```diff
@@ -0,0 +1,75 @@
+use crate::helpers::constants::{
+    ACCOUNT, ACCOUNT_FILE_PATH, CASM_CONTRACT_PATH, NETWORK, SIERRA_CONTRACT_PATH, URL,
+};
+use camino::Utf8PathBuf;
+use cast::{get_account, get_network, get_provider};
+use starknet::accounts::{Account, SingleOwnerAccount};
+use starknet::contract::ContractFactory;
+use starknet::core::types::contract::{CompiledClass, SierraClass};
+use starknet::core::types::FieldElement;
+use starknet::providers::jsonrpc::HttpTransport;
+use starknet::providers::JsonRpcClient;
+use starknet::signers::LocalWallet;
+use std::sync::Arc;
+
+pub fn account(
+    provider: &JsonRpcClient<HttpTransport>,
+) -> SingleOwnerAccount<&JsonRpcClient<HttpTransport>, LocalWallet> {
+    get_account(
+        ACCOUNT,
+        &Utf8PathBuf::from(ACCOUNT_FILE_PATH),
+        provider,
+        &get_network(NETWORK).unwrap(),
+    )
+    .unwrap()
+}
+
+pub async fn declare_simple_balance_contract() {
+    let provider = get_provider(URL).unwrap();
```

Comment:
I do not think we should be using `unwrap` as much as you are, even in tests. Can you replace them at least with `expect`? Even this tiny error message can sometimes be helpful, you don't need to dig into the code, you sometimes just see the message and know what is wrong right away.

Reply:
What happens when a test panics? All tests break or just one fails?

Reply:
afaik just one

Reply:
https://github.com/software-mansion/protostar/pull/2160/commits/d7c616450e7a4dc87ba63e50d23bc1f1ae4516de